rephrase_chempile/rephrase.py

```diff
@@ -0,0 +1,256 @@
+from dotenv import load_dotenv
+from openai import OpenAI
+from datasets import get_dataset_config_names, load_dataset, Dataset, DatasetDict
+import random
+import tiktoken
+import os
+
+load_dotenv("../.env", override=True)
+
+client = OpenAI()
+
+
+SIMPLE_PROMPT = """Given the text below:
+
+{document}
+
+Generate exactly one question and its corresponding answer based solely on the content.
+
+Format your response as:
+Question: [your question]
+Answer: [your answer]"""
+
+HARD_PROMPT = """Rephrase the text below into a scholarly dialogue using these requirements:
+
+1. Structure as multiple alternating "Question:" and "Answer:" pairs
+2. Use esoteric vocabulary and complex syntax suitable for academics
```

Comment:
To increase diversity of the results, I would consider randomly sampling between a few synonyms